hudi-cli/src/main/java/org/apache/hudi/cli/utils/InputStreamConsumer.java

```diff
@@ -46,7 +47,7 @@ public void run() {
         LOG.info(line);
       }
     } catch (IOException ioe) {
-      LOG.severe(ioe.toString());
+      LOG.error(ioe.toString());
```

Comment:
Can we simplify it with `LOG.error(ioe);`?

Reply:
> Can we simplify it with `LOG.error(ioe);`?

logging  -> SLF4J
FINEST  -> TRACE
FINER   -> DEBUG
FINE    -> DEBUG
CONFIG  -> INFO
INFO    -> INFO
WARNING -> WARN
SEVERE  -> ERROR

Reply:
I have no doubt about `SEVERE -> ERROR`. What I mean is that can we remove ioe ~.toString()~?

Reply:
yes, you are right, The method `ToString()` can remove.